pcloudcv/requirements.txt

```diff
@@ -3,18 +3,16 @@ numpy
 matplotlib
 Pillow
 h5py
-redis >= 2.7.6
-colorama >= 0.2.5
-poster >= 0.8.1
-CherryPy >= 3.2.4
+redis>=2.7.6
+colorama>=0.2.5
+poster>=0.8.1
+CherryPy>=3.2.4
```

Comment:
Leaving spaces between the package name and its version gives an error while installing since it assumes the version as another package and tries to install it. So, I have removed the spaces.